docs/GettingStartedDocs/buildandsign.md

```diff
@@ -97,6 +97,15 @@ NumHeapPages=1024
 NumStackPages=1024
 NumTCS=1
 ```
+Additionally, for platform that supports for Key sharing and separation (KSS),
+the following properties can be used:
```

Comment:
```suggestion
the following additional properties can be used:
```

Also explain what happens if these are included and then the enclave is run on a non KSS machine.   Will it fail to load?  Will the properties simply be ignored?  I.e., does the developer need to compile the enclave twice, once for KSS and once without, to run on both types of platforms?